### ISSUE TITLE
Allow the card images to be downloaded as separate packs

### ISSUE THREAD

**Issue by scottchiefbaker** [danieljbrooks/draftsim-launch#16]
We should move the card images out of this git repo, and provide them another method (as zip files?) to users to download. That way if someone wants to host a draftsim instance but ONLY wants to draft one set they just download the card set they want.

We could modify the PHP to look in `/cards/` to see what sets are available and draw the header appropriately.

**Comment by danieljbrooks** [comment]
I could see moving the images to a new location to save transfer space,
although keeping in the repository keeps all the files in the same place.

Most users are going to interact with this through the draftsim website

**Comment by arxanas** [comment]
Good lord, yes, cloning the repo took ages, I assume because the images are checked in. (It's also not really copyright-legal to bundle them with the repo, is it?)

I definitely think that we should do the following:

1. Have automated set listing, as the current way of listing them manually doesn't scale. (Ideally, I think any directory structure should be called `sets` rather than `cards`.) 
2. Not bundle cards with the repo, since that tales ages to download and is of dubious legality.

Where did you get the images from? I had an idea to proxy the images to a PHP page which would download the image if it didn't already exist. This solves the bundling issue, as well as downloading them from somewhere.

**Comment by danieljbrooks** [comment]
Good points Waleed.

The initial design of the site was put together with two goals in mind:
-Good user experience
-Platform to develop accurate quantitative model for limited

Image files are currently hosted in the Github directory because the
initial Github files were copied from the website, which has image files in
a directory. A redesign is certainly possible.

The current images were obtained from the Cockatrice image folder, although
those have now been switched out for JPEG's, which are a factor of 10
smaller.

There's actually a really interesting implementation of card data available
from: http://mtgjson.com/ . This could be useful for scaling to multiple
sets.

**Comment by arxanas** [comment]
@danieljbrooks Of course, http://mtgjson.com is where Cockatrice's Oracle tool gets its data from :P

**Comment by danieljbrooks** [comment]
On the topic of images, I've recently found gatherer extractor to be a good tool for getting .jpg images from gatherer

http://www.mtgsalvation.com/forums/magic-fundamentals/digital-magic-and-mtg-software/337224-mtg-gatherer-extractor-v3-18-database-pics
